src/components/views/auth/RegistrationForm.js

```diff
@@ -565,8 +607,8 @@ module.exports = React.createClass({
                         {this.renderEmail()}
                         {this.renderPhoneNumber()}
                     </div>
-                    {_t("Use an email address to recover your account.") + " "}
-                    {_t("Other users can invite you to rooms using your contact details.")}
+                    <p>{_t("Use an email address to recover your account.")}</p>
+                    { this.renderBindEmail() }
```

Comment:
This should probably only be shown if you actually fill in the e-mail box otherwise it makes no sense. (the email box has `(optional)` written in it)

Reply:
Yeah, although then it would have to appear after you'd entered an email address, which is a bit weird. IMO people will probably get the idea if something says it'll use their email address for something an they don't enter one.